### DISCUSSION TITLE
Option to search Unddit or similar api for removed comments

### DISCUSSION THREAD

**Discussion by bonn2** [Docile-Alligator/Infinity-For-Reddit#1121] [Completed Ideas] [answered]
It would work similar to the Redgifs backup, when Reddit serves a deleted comment infinity would search [https://unddit.com](https://unddit.com/) then display it instead along with a small badge or something to show that the comment was removed. This could also have the option to operate like a spoiler region. 
It appears that unddit uses [https://pushshift.io/](https://pushshift.io/) so that may be the correct api to query for it.

**Comment by moon-cacke** [comment]
Services that can be used for that : Unddit, Reveddit, Resavr, Webarchive, Google-cache...
A simple implementation could be a button on the toolbar that open a drop list with web links to those services

**Comment by Wladefant** [comment, marked as answer]
Hey, we are already using the pushshift and reveddit api. To see the removed comment just tap on the 3 dots>See removed comment 😄
